src/database/mongo.js

```diff
@@ -28,7 +29,7 @@ mongoModule.questions = [
 	{
 		name: 'mongo:host',
 		description: 'Host IP or address of your MongoDB instance',
-		default: nconf.get('mongo:host') || '127.0.0.1',
+		default: nconf.get('mongo:host') || ip.address(),
```

Comment:
The default should be to assume the database is installed on the local machine, and to use the shorthand `127.0.0.1`. If the default were the external IP of the server, and the installation was moved, the config would no longer be correct.